lib/src/validationsDispatcher.ts

```diff
@@ -59,20 +61,21 @@ export class ValidationDispatcher {
   }
 
   fireAllFieldsValidations(
-    vm: Object,
+    vm: any,
+    vmKeys: string[],
```

Comment:
Ok, thanks. It was a bit confusing for me when I read `vm` and then `vmKeys`.

Maybe some rename like: `vmKeysWithValidation`, `vmKeysToValidate`, `fieldsToValidate`?